### ISSUE TITLE
[Core] - UH DK Pets Bugged

### ISSUE THREAD

**Issue by Erayde** [GuabinaCore/WoWKoi#3894]
Hi Uh Dk solo target lego-talents-stats are correct but i cant do solo target damage , when i switch to area damage talent i hit more than solo target talent in warcraft logs is this not normal In short, the solo target build that I buff the zombies does not work because the damage of the zombies is extremely low.

![Screenshot_145](https://user-images.githubusercontent.com/94382034/197416440-76985676-84d7-4755-916c-b43604ba499c.png)
bis solo target build
with lego talents and stats
![Screenshot_144](https://user-images.githubusercontent.com/94382034/197416625-9360b198-400d-4740-9671-4392558bfe16.png)

m+ aoe +si
ngle  karma  legos talents
I tried on other legion servers, the only problem is that the zombies hit less, especially the Dark Transformation ability.
ı pres this skill my zombie change to cleave skill to Vile Gas
this vile gas doing high aoe and solo target dps
in other server
basicly normally
but here cant work truly
30-60k damage one hit
with 30-60k str and  ½100 mastery
On other servers it hits between 110-290k with 30k Strenght
with ½82 mastery
In summary, things like gear stat don't buff zombies much, or in general the damage done by my zombies is limited.
can u check this if possible
